README.md

```diff
@@ -1,2 +1,69 @@
-# windup-operator
-Windup Operator for Openshift-Kubernetes
+# windup-operator project
+
+This project creates an Operator to deploy Windup/MTA on an Openshift cluster ( Kubernetes one in future releases)
+
+It consists on a series of yaml files to deploy the operator (CRD, role, rolebinding, serviceaccount) and the native artifact compiled using Quarkus and GraalVM
+
+At this moment the operator reacts to creation of the Windup Custom Resource and will deploy all needed objects ( deployments, ingress, services, persistent volumes)
+
+## Building and pushing the Java code manually in native mode
+1. The following command will use the configuration in the `application.properties` file :  
+```
+quarkus.container-image.registry=quay.io
+quarkus.container-image.group=windup
+quarkus.container-image.name=${quarkus.application.name}-native
+quarkus.container-image.tag=latest
+quarkus.kubernetes.service-type=load-balancer
+quarkus.kubernetes.image-pull-policy=never
+quarkus.container-image.builder=docker
+namespace=mta
```

Comment:
Worth updating the sample values provided in the README with latest properties and values.
For example `namespace=mta` should be removed